docs/src/installation.rst

```diff
@@ -13,4 +13,5 @@ Install from source
   cd scikit-matter
   python setup.py install --user
 
-You're ready to import skmatter from your code! Have a look at the :ref:`api-reference` for how to use the code.
+You're ready to import skmatter from your code! Have a look at the :ref:`api-reference`
+for how to use the code.
```

Comment:
```suggestion
You're ready to import skmatter from your code! Have a look at the :ref:`api-reference` for how to use the code.
```

Reply:
We can put this in one line but should not try to use 88 characters also in the docs? rst does not care about a new line.